lib/axon_onnx/deserialize.ex

```diff
@@ -2018,85 +2018,44 @@ defmodule AxonOnnx.Deserialize do
   defp recur_nodes(
          %Node{
            op_type: "Resize",
-           input: input,
+           input: [in_layer_name | rest_of_inputs],
            attribute: attrs,
-           output: output
+           output: [out_layer_name]
          },
          {axon, params, used_params}
        ) do
     # op_type spec: https://github.com/onnx/onnx/blob/main/docs/Operators.md#resize
 
-    [in_layer_name | rest_of_inputs] = input
-    inp = input!(in_layer_name, axon, params, used_params)
-
-    layer_inputs =
-      inp
-      |> Axon.get_inputs()
-      |> Map.new(fn {k, v} -> {k, Nx.broadcast(0.0, v)} end)
-    input_shape = Axon.get_output_shape(inp, layer_inputs)
-
-
-    [_roi, scales, sizes] =
-      case rest_of_inputs do
-        # all are optional
-        # only one of 'scales' and 'sizes' can be specified
-        ["", scales] ->
-          scales = constant!(scales, axon, params, used_params) |> Nx.to_flat_list()
-          [nil, scales, nil]
-
-        ["", _, sizes] ->
-          sizes = constant!(sizes, axon, params, used_params) |> Nx.to_flat_list()
-          [nil, nil, sizes]
-
-        [_roi, _sc] ->
-          raise ArgumentError, "unsupported resize, roi is not implemented yet"
-        [_roi, _sc, _si] ->
-          raise ArgumentError, "unsupported resize, roi is not implemented yet"
-      end
-
-    output_shape =
-      case [scales, sizes] do
-        [nil, nil] ->
-          raise ArgumentError, "Invalid Resize, one of 'scales' and 'sizes' MUST be specified"
-
-        [scales, nil] ->
-          input_shape
-          |> Tuple.to_list()
-          |> Enum.zip_with(scales, fn
-            nil, _  -> nil
-            is, sc -> trunc(is * sc)
-          end)
-
-        [nil, sizes] ->
-          sizes
-
-      end
-
     %{"mode" => mode} = options!(attrs)
-
     method =
       case mode do
-        "nearest" ->
-          :nearest
-
-        "linear" ->
-          :linear
+        "linear" -> :bilinear
+        _mode -> raise ArgumentError, "only linear is implemented"
+      end
 
-        "bilinear" ->
-          :bilinear
+    scales =
+      case rest_of_inputs do
+        ["", scales] -> input!(scales, axon, params, used_params)
+        _ -> raise ArgumentError, "only scales is implemented"
+      end
 
-        "cubic" ->
-          :cubic
+    # resize function
+    fun = fn input, scales, _opts ->
+      # this will return a Tensor, but we need a tuple
+      #   we are in Defn env now, so Nx.to_flat_list() does not work
+      output_size = input
+        |> Nx.shape()
+        |> Tuple.to_list()
+        |> Nx.tensor()
+        |> Nx.multiply(scales)
+        |> IO.inspect()
+
+      Axon.Layers.resize(input, size: output_size, method: method)
```

Comment:
This fails, since `Axon.Layers.resize` wants to have a Tuple instead of a Tensor (`output_size`)